hack/move-upstream.sh

```diff
@@ -18,6 +18,8 @@ repo="${UPSTREAM_REPO:-k8s.io/kubernetes}"
 package="${UPSTREAM_PACKAGE:-pkg/api}"
 
 patch="${TMPDIR}/patch"
+rm -rf "${patch}"
```

Comment:
should probably be mktemp, although that sucks because it makes it hard to find the patches.